.travis.yml

```diff
@@ -2,21 +2,22 @@ language: rust
 rust:
   - stable
   - nightly
-
+os:
+  - linux
+  - osx
+  - windows
 env:
-  - NODE_ENV=ci
-  - NODE_ENV=ci-staging
-cache: yarn
-
+  - NODE_ENV=ci YARN_GPG=no
+  - NODE_ENV=ci-staging YARN_GPG=no
+cache: cargo
 before_install:
   - rustup target add wasm32-unknown-unknown
-  - nvm install 12
-  - nvm use 12
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then HOME="/c/Users/travis" && export NVS_HOME=$ProgramData/nvs && git clone --single-branch https://github.com/jasongin/nvs $NVS_HOME && source $NVS_HOME/nvs.sh && nvs --version && nvs add 12 && nvs use 12; fi
+  - if [[ ! "$TRAVIS_OS_NAME" == "windows" ]]; then nvm install 12 && nvm use 12; fi
   - npm install -g yarn
-
-install:
-  - yarn install
-
+  - yarn
+git:
+  autocrlf: false
```

Comment:
Do we need this?  I am assuming this is for windows git? My windows git setup works without this.

Reply:
It looks like the commit right before the last one [failed](https://github.com/nearprotocol/create-near-app/pull/165/commits) with [a bunch of `CRLF` errors](https://travis-ci.com/github/nearprotocol/create-near-app/jobs/309464923)

Reply:
Yes, Travis tests will fail for Windows without this.